coconut/cmd/environment_create.go

```diff
@@ -46,21 +46,21 @@ A valid workflow template (sometimes called simply "workflow" for brevity) must
 
 Workflows and tasks are managed with a git based configuration system, so the workflow template may be provided simply by name or with repository and branch/tag/hash constraints.
 Examples:
- * ` + "`coconut env create -w myworkflow`" + ` - loads workflow ` + "`myworkflow`" + ` from default configuration repository at HEAD of master branch
- * ` + "`coconut env create -w github.com/AliceO2Group/MyConfRepo/myworkflow`" + ` - loads a workflow from a specific git repository, HEAD of master branch
- * ` + "`coconut env create -w myworkflow@rev`" + ` - loads a workflow from default repository, on branch, tag or revision ` + "`rev`" + `
- * ` + "`coconut env create -w github.com/AliceO2Group/MyConfRepo/myworkflow@rev`" + ` - loads a workflow from a specific git repository, on branch, tag or revision ` + "`rev`" + `
-
-For more information on the %s workflow configuration system, see documentation for the ` + "`coconut repository`" + ` command.`, product.PRETTY_SHORTNAME, product.PRETTY_SHORTNAME),
-	Run:   control.WrapCall(control.CreateEnvironment),
+ * `+"`coconut env create -w myworkflow`"+` - loads workflow `+"`myworkflow`"+` from default configuration repository at HEAD of master branch
+ * `+"`coconut env create -w github.com/AliceO2Group/MyConfRepo/myworkflow`"+` - loads a workflow from a specific git repository, HEAD of master branch
+ * `+"`coconut env create -w myworkflow@rev`"+` - loads a workflow from default repository, on branch, tag or revision `+"`rev`"+`
+ * `+"`coconut env create -w github.com/AliceO2Group/MyConfRepo/myworkflow@rev`"+` - loads a workflow from a specific git repository, on branch, tag or revision `+"`rev`"+`
 
+For more information on the %s workflow configuration system, see documentation for the `+"`coconut repository`"+` command.`, product.PRETTY_SHORTNAME, product.PRETTY_SHORTNAME),
+	Run: control.WrapCall(control.CreateEnvironment),
 }
 
 func init() {
 	environmentCmd.AddCommand(environmentCreateCmd)
 
+	environmentCreateCmd.Flags().StringP("configuration", "c", "", "high-level configuration payload to be loaded for the new environment")
+	environmentCreateCmd.MarkFlagRequired("configuration")
 	environmentCreateCmd.Flags().StringP("workflow-template", "w", "", "workflow to be loaded in the new environment")
-	environmentCreateCmd.MarkFlagRequired("workflow-template")
```

Comment:
We must be able to run
* with `-w` but no `-c` (behaviour until now)
* with `-c` but no `-w`
* with `-c` and `-w` and possibly other parameters.

IOW: if `-c`, all params are optional, else `-w` is required.
Could be that Cobra already offers this logic instead of plain `MarkFlagRequired` (check docs please).